x-pack/plugins/maps/public/classes/styles/vector/vector_style.tsx

```diff
@@ -99,6 +103,8 @@ export interface IVectorStyle extends IStyle {
   isTimeAware(): boolean;
   getPrimaryColor(): string;
   getIcon(showIncompleteIndicator: boolean): ReactElement;
+  getIconMeta(symbolId: string): { svg: string; label: string; iconSource: ICON_SOURCE };
```

Comment:
getIconMeta implementation returns undefined, {}, and { svg: string; label: string; iconSource: ICON_SOURCE }. Update this type to `{ svg: string; label: string; iconSource: ICON_SOURCE } | undefined`